pkgs/applications/editors/pulsar/default.nix

```diff
@@ -0,0 +1,169 @@
+{ lib
+, stdenv
+, git
+, runtimeShell
+, fetchzip
+, wrapGAppsHook
+, glib
+, gtk3
+, atomEnv
+, xorg
+, libxkbcommon
+, hunspell
+, hunspellDicts
+, useHunspell ? true
+, languages ? [ "en_US" ]
+, withNemoAction ? true
+, makeDesktopItem
+, copyDesktopItems
+, makeWrapper
+}:
+
+let
+  pname = "Pulsar";
+  version = "1.103.0";
+
+  sourcesPath = {
+    x86_64-linux = {
+      tarname = "Linux.${pname}-${version}.tar.gz";
+      hash = "sha256-O+mekV2h3mxRPYpIrpoPHQyRDuXgl+En8n8u2yBG8TQ=";
```

Comment:
this derivation has two hashes, i missed that when making the update script. it would need to be adjusted to account for this.